actions/contacts/reject.go

```diff
@@ -0,0 +1,32 @@
+package contacts
+
+import (
+	"net/http"
+
+	"github.com/bitcoin-sv/spv-wallet/server/auth"
+	"github.com/gin-gonic/gin"
+)
+
+// reject will reject contact request
+// Reject contact godoc
+// @Summary		Reject contact
+// @Description	Reject contact. For contact with status "awaiting" delete contact
+// @Tags		Contact
+// @Produce		json
+// @Param		paymail query string true "paymail"
+// @Success		200
+// @Router		/v1/contact/reject [PATCH]
```

Comment:
Hmm, I think that something doesn't fit together:
`// @Param		paymail query string true "paymail" `

Here you introduced the query param, but in `routes.go` it looks that it is rather a path param:
`contactGroup.PATCH("/reject/:paymail", action.reject)`

The second thing is that the endpoint should be named as:
`/v1/contact/rejected`
and
`/v1/contact/accepted`

So base on our previous team discussion:
<img width="332" alt="image" src="https://github.com/bitcoin-sv/spv-wallet/assets/48433067/07cc314f-8e1d-45c1-955e-ef9df67e249f">

Reply:
### To sum it up and be consistent with our arrangements:

1. Rename route paths to:
`/v1/contact/rejected/:paymail`
and
`/v1/contact/accepted/:paymail`

2. Stick with path params in swagger:
`// @Param		paymail ~~query~~ string true "paymail"`
`// @Param		paymail path string true "paymail"`

and add this param to the route in swagger:
`// @Router		/v1/contact/rejected/{paymail}`
`// @Router		/v1/contact/accepted/{paymail}`

Please regenerate swagger after those changes.

Reply:
I think it in the scope of https://bitcoinassociation.atlassian.net/jira/software/c/projects/BUX/boards/22?selectedIssue=BUX-648

Reply:
@arkadiuszos4chain I would vote to do this now, why to not have properly adjusted endpoints if it's a fresh PR?